MeetupPlatformApi/Features/Meetups/GetMeetup/ContactInfoDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace MeetupPlatformApi.Features.Meetups.GetMeetup;
+
+public class ContactInfoDto
```

Comment:
Create class `ContactInfoDto` into `MeetupInfoDto`. Example is in `RegistrationResultDto`